lib/driver.js

```diff
@@ -207,9 +208,12 @@ function connectBrowser(context, callback) {
       callback(null);
       return;
     }
-    request(context, 'POST', '/timeouts/async_script', {
-      ms : context.timeout || 10000
-    }, function (err) {
+
+    var timeouts = context.timeouts || {
+      script: context.timeout || 10000
```

Comment:
Now that the same timeout value is used, I'd prefer to go back to the previous configuration style to only have a single "timeout" value.

Reply:
This is a leftover and not used anymore. I removed it.